ghost-cat-app/src/app/View/image-thumbnails-page/image-thumbnails-page.component.html

```diff
@@ -1 +1,39 @@
-<p>image-thumbnails-page goes here</p>
+<!DOCTYPE html>
```

Comment:
I'm pretty sure that for an angular component you don't need this `<!DOCTYPE html>` tag. You probably won't need the `<html>`, `<head>`, or `<body>` tags either